indexs.js

```diff
@@ -1,110 +1,45 @@
+const express = require('express')
+const mongoose = require('mongoose')
+const User = require('./user');
+let url = "mongodb+srv://prateek:java1234@cluster0.vkgzh.mongodb.net/nodedb?retryWrites=true&w=majority"
 
-var MongoClient = require('mongodb').MongoClient;
-var url = "mongodb+srv://prateek:java1234@cluster0.vkgzh.mongodb.net/nodedb?retryWrites=true&w=majority";
+mongoose.connect(url, { useNewUrlParser: true, useUnifiedTopology: true })
 
-const client = new MongoClient(url);
+var app = express()
 
-const dbName = "nodedb";
+const bcrypt = require('bcrypt');
+const saltRounds = 10;
 
-async function run() {
-    try {
-        await client.connect();
-        console.log("Connected correctly to server");
-        const db = client.db(dbName);
-        const userData = db.collection("users");
-        const userDetails = db.collection("usersProfile");
-        async function create(user,profile){
-            const usersData = await userData.insertOne(user);
-            profile.userId = usersData.insertedId;
-            const profilesData = await userDetails.insertOne(profile)
-            return 'User created succesully'
+const port = process.env.PORT || 3000;
+app.use(express.json());
 
-        }
 
-    
-        let users = [{
-            "firstname": "vivek",
-            "lastname": "dagur",
-            "age": 25,
-            "email": "vivekdagur8@gmail.com",
-            "password": "react123",
-            "dob": "222",
-            "mobile": "1w1"
-        },
-        {
-            "firstname": "prateek",
-            "lastname": "dagur",
-            "age": 23,
-            "email": "prateekdagur8@gmail.com",
-            "password": "node123",
-            "dob": "222",
-            "mobile": "1w1"
-        },
-        {
-            "firstname": "ankur",
-            "lastname": "paliwal",
-            "age": 24,
-            "email": "anku78@gmail.com",
-            "password": "ank123",
-            "dob": "222",
-            "mobile": "1w1"
 
+app.post('/user/register', async (req, res) => {
 
-        },
-        {
-            "firstname": "divyanshu",
-            "lastname": "sharma",
-            "age": 26,
-            "email": "dv3456@gmail.com",
-            "password": "dvw123",
-            "dob": "222",
-            "mobile": "1w1"
-        },
+    try {
 
-        {
-            "firstname": "kapil",
-            "lastname": "dagur",
-            "age": 27,
-            "email": "kapilch@gmail.com",
-            "password": "node123",
-            "dob": "222",
-            "mobile": "1w1"
-        }
-    ]
+        const hash = bcrypt.hashSync(req.body.password, saltRounds);
```

Comment:
use salt for creating **hash** and store that salt in database (as mentioned in last PR review)